frontend/app/module/data_entry/page/DataEntryHomePage.tsx

```diff
@@ -46,13 +48,17 @@ export function DataEntryHomePage() {
           <WorkStationNumber>16</WorkStationNumber>
         </section>
       </header>
-      {showDataEntrySavedAlert && (
+      {dataEntryDone && (
         <Alert type="success" onClose={closeDataEntrySavedAlert}>
           <h2>Je invoer is opgeslagen</h2>
           <p>
             Geef het papieren proces-verbaal terug aan de coördinator.
-            <br />
-            Een andere invoerder doet straks de tweede invoer.
+            {showFirstDataEntrySavedAlert && (
+              <>
+                <br />
+                Een andere invoerder doet straks de tweede invoer.
+              </>
+            )}
           </p>
```

Comment:
Can you add the translations for the alert (this might be done in #660, but will need to be updated there then based on these changes.

Reply:
I've merged #660, once I've resolved the conflicts the translations should be fixed.

Reply:
Done in e717ba0c01ce99b2f6bef09786d01998e2a2b5b9